### PR TITLE
Add Jami (new name / successor of cx.ring.Ring)

### PR DESCRIPTION
**Pull request by alexander255**
As per the discussion on https://github.com/flathub/cx.ring.Ring/issues/10, I'm submitting this new version of Ring as a new application here. The contents of this branch are just a snapshot of the current Jami Flatpak version that may be found here: https://gitlab.com/ntninja/net.jami.Jami – I fully intend to force-push over whatever repository is created for me with that version to retain the previous commit history. Hopefully this is not a problem.
Legally, there should be no difference between this and cx.ring.Ring as I only updated and removed packages in the new image.

### CODE REVIEW

File: net.jami.Jami.json [hunk 1]
```diff
@@ -0,0 +1,179 @@
+{
+	"app-id": "net.jami.Jami",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "3.28",
```
Comment (LeandroStanger) [inline net.jami.Jami.json]: Updte runtime to 3.32

**Comment by LeandroStanger** [comment]
bot, build net.jami.Jami

File: net.jami.Jami.json (hunk 1 above)
Comment (alexander255) [reply net.jami.Jami.json]: I cannot at the moment, as the application crashes when I do that and I still wasn't able to download `org.gnome.Sdk.Debug` to debug this (503 Service Unavailable). See also my last [commit](https://gitlab.com/ntninja/net.jami.jami/commit/63a611df12a8eddbb4acc9d562a85e2c79faeb8d) regarding this and discussions on possibly related issues: https://github.com/flathub/cx.ring.Ring/issues/2 & https://github.com/flathub/cx.ring.Ring/issues/3.

**Comment by ovari** [comment]
@LeandroStanger how can [Jami](https://jami.net/) be:
1. added to [Flathub](https://flathub.org/); and
2. hosted at https://github.com/flathub/net.jami.Jami

Thank you

**Comment by barthalion** [comment]
As we currently don't handle renames, you will need to add EOL notice to the old repo when it gets merged and ask an admin to archive it.

File: io.qt.qt5-base.json [hunk 2]
```diff
@@ -0,0 +1,87 @@
+{
```
Comment (barthalion) [inline io.qt.qt5-base.json]: It was probably addressed during initial review but I will ask anyway: it requires both Qt and full GNOME runtime?

Comment (alexander255) [reply io.qt.qt5-base.json]: It doesn't require the full GNOME runtime, but it's a GTK + some GNOME stuff GUI app, yes. Qt is required (at least for the foreseeable future) for `libringclient` which contains everything non-GUI that is shared between different client implementations. The dependency on `Qt5Gui` should be gone very soon however, it's only used in one place to load, scale and save avatar images.

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/net.jami.Jami

Let me know when you add EOL notice to the old repo. Thanks!

**Comment by alexander255** [comment]
@barthalion: I think we may have a situation here: Apparently the id `net.jami.Jami` was based on a misunderstanding with upstream, it should have been `net.jami.client.gnome` (since there are also other Jami clients, such as the third-party KDE one and also the official Windows, macOS, iOS and Android clients). Would it be hard to change the ID at this point, or would it be better not to? I see the app is not listed on https://flathub.org/apps/details/net.jami.Jami yet, so I guess?

**Comment by Nokia808** [comment]
I'm already installed it ! It is already listed & appearing in my GNOME software center.

**Comment by alexander255** [comment]
@Nokia808: Thanks, I now see it too. I guess this settles the ID question then.

**Comment by Nokia808** [comment]
It is up to you, but I feel that name "net.jami.client.gnome" is better. What prevent renaming ??

**Comment by alexander255** [comment]
@Nokia808: It's not up to me, the Flathub admins need to do it. Of course, now that the package is live under the wrong name, we'd probably have to deprecate the package and push yet another *new* package because Flatpak (the client, not the service) afaik cannot handle changes in the app-id (which, after all, is not supposed to change).

@barthalion: I'd appreciate an official comment on these assumptions of mine.

**Pull request closed without merging**